collections/sea-ice-index.yaml

```diff
@@ -44,7 +44,12 @@ Resources:
     Role: processor
     Type: byoc-10549890-13bb-4ec5-8ae0-df387bd0b785
     CollectionId: 10549890-13bb-4ec5-8ae0-df387bd0b785
-    Primary: true  
+    Primary: true
+WMTS:
+  - href: https://creodias.sentinel-hub.com/ogc/wmts/2a0231a2-5c91-4d3b-a911-ff55711aa983
```

Comment:
I didn't manage to test this (see my question in the Sentinel chat). Did this work for you when you tested it?

Reply:
WMTS link object for this collection was removed until it's fixed. See [issue](https://git.sinergise.com/sentinel-core/java/-/issues/2470).